packages/-ember-data/tests/helpers/qunit-asserts/assert-assertion.ts

```diff
@@ -0,0 +1,86 @@
+import QUnit from 'qunit';
+import { checkMatcher } from './check-matcher';
+import { DEBUG } from '@glimmer/env';
+import RSVP from 'rsvp';
+
+let HAS_REGISTERED = false;
+
+interface AssertSomeResult {
+  result: boolean;
+  actual: string;
+  expected: string;
+  message: string;
+}
+interface AssertNoneResult {
+  result: boolean;
+  actual: string;
+  expected: '';
+  message: string;
+}
+
+function expectAssertion(message: string, matcher: string | RegExp): AssertSomeResult {
```

Comment:
Lets call this method something different, its pretty confusing to have both `QUnit.assert.expectAssertion` and this inline function with the same name but quite different.

I'm thinking something like `verifyAssertions`...